ci/dev/try_builders.json

```diff
@@ -28,7 +28,8 @@
       {
          "name":"Linux Web Engine",
          "repo":"engine",
-         "enabled": true
+         "enabled": true,
+         "run_if": ["DEPS", "lib/web_ui/**", "web_sdk/**"]
```

Comment:
Should we add `e2etests`, `tools` (which contains `clone_flutter.sh`), `ci` (which runs some linters and analysis), and `flutter_frontend_server`?

Reply:
thanks for adding these items. 

btw for clarification `clone_flutter.sh` is only used within felt right now, I will also remove it soon. A new script is used from inside the LUCI recipes. The new script we are using is [here](https://github.com/flutter/engine/blob/master/tools/configure_framework_commit.sh). The reason is we can't use the previous code we used for determining the branch as we did in cirrus. We need to pass the branch to properties.

For example usage from the [recipe](https://flutter.googlesource.com/recipes/+/refs/heads/master/recipes/engine/web_engine_framework.py#114)